inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -706,7 +707,7 @@ std::shared_ptr<CNNNetworkImpl> convertFunctionToICNNNetwork(const std::shared_p
     for (const auto &layer : nodes)
         op_names.insert(layer->get_name());
 
-    bool keep_constants = ::ngraph::op::util::has_op_with_type<::ngraph::op::FakeQuantize>(graph);
+    bool keep_constants = keep_constant_inputs; // ::ngraph::op::util::has_op_with_type<::ngraph::op::FakeQuantize>(graph);
```

Comment:
It is not correct. Please set keep constants if bool flag is true or FakeQuantize operation was found.

Reply:
This logic was moved to kmb-plugin. But I think that it is possible to place it here.

Reply:
It is not correct because another plugins also depends on this logic.
CPU plugin for example also needs this logic.

Reply:
Please look at new version with if-else logic.